src/applications/disability-benefits/all-claims/components/ObjectViewField.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+const ObjectViewField = props => {
```

Comment:
This is app code, so don't feel any obligation to listen to me on this one, but I had a recommendation anyhow. :smile: 

Consider using a less generic name for this. I find either naming the pattern (`SimpleDisplayReviewField`) or naming the use (`ConditionReviewField`) works pretty well.

Reply:
I have no opinion on this, but I can't think of a good name. Any preference?

Reply:
:man_shrugging: I'd probably go with `ConditionReviewField` myself, but it's your call, of course. :smile: